Short-Answer/Algorithms_Answers.md

```diff
@@ -3,13 +3,15 @@
 ## Exercise I
 
 a)
-
+0(n)
+This algorithms run time is 0(n) because its a while loop that'll grow at the same rate depending on what n is.
 
 b)
-
-
+0(log n)
+This algorithms run time is 0(log n) because it first takes in a for loop then a while loop where j doubles until it reaches n.
```

Comment:
You have half of the solution here:

In the while loop, j doubles to reach n, so that is `O(logn)`. But, the outer loop also has a time complexity of `O(n)`.

If the loops are nested, we multiply the time complexities together for a final complexity of `O( n logn )`